sdk/go/common/util/logging/log.go

```diff
@@ -141,12 +142,23 @@ func (f *nopFilter) Filter(s string) string {
 	return s
 }
 
-type replacerFilter struct {
-	replacer *strings.Replacer
+type ahoCorasickFilter struct {
+	replacement string
+	replacer    aho_corasick.Replacer
```

Comment:
ordering nit: 
would you mind moving this struct definition and its filter method closer to (probably right above) `func CreateFilter`?
It's not otherwise obvious where this is used, especially because there's a global filter defined and used right below this.